lib/studies/reply/mail_view_page.dart

```diff
@@ -1,14 +1,98 @@
 import 'package:flutter/material.dart';
+import 'package:gallery/studies/reply/colors.dart';
 
 class MailViewPage extends StatelessWidget {
   const MailViewPage({Key key}) : super(key: key);
   @override
   Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
+
     return Scaffold(
-      body: Center(
-        child: OutlineButton(
-          onPressed: () => Navigator.pop(context),
-          child: const Text('Back'),
+      body: SafeArea(
+        child: Padding(
+          padding:
+              const EdgeInsetsDirectional.only(top: 42, start: 20, end: 20),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Row(
+                children: [
+                  Expanded(
+                    child: Text(
+                      'Package shipped!',
+                      style: textTheme.headline4.copyWith(height: 1.1),
+                    ),
+                  ),
+                  IconButton(
+                    icon: const Icon(Icons.keyboard_arrow_down),
+                    onPressed: () {
+                      Navigator.pop(context);
```

Comment:
Why pop when this is pressed?

Reply:
This pops the mail view page, and sends you back to then main inbox.